workflow/rocoto/tasks.py

```diff
@@ -205,10 +206,9 @@ def get_resource(self, task_name):
         else:
             # Memory is not required
             memory = task_config.get(f'memory_{task_name}', None)
-
-        if scheduler in ['pbspro']:
-            if task_config.get('prepost', False):
-                memory += ':prepost=true'
+            if scheduler in ['pbspro']:
+                if task_config.get('prepost', False):
+                    memory += ':prepost=true'
```

Comment:
How is this change related to AWS?

Reply:
I do not think so. This should be from sync with EMC repo.
I do not remember make this change, and I do not think I have test anythin related to "pbspro".

Reply:
copy over from EMC repo again, only change the native = None part.